Fakebook.Posts/Fakebook.Posts.Domain/Models/Post.cs

```diff
@@ -1,8 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
 
-namespace Fakebook.Posts.Domain {
-    public class Post {
+namespace Fakebook.Posts.Domain.Models
+{
+    public class Post
```

Comment:
this isn't very OOP. surely, e.g., null isn't acceptable for useremail, so shouldn't the constructor enforce providing a valid value? classes like this, we want constructors to enforce providing required data, and probably the setters don't need to be public for several of these things, unless a user can change his email or something.